plugins/vpc-shared-eni/plugin/commands.go

```diff
@@ -89,6 +94,19 @@ func (plugin *Plugin) Add(args *cniSkel.CmdArgs) error {
 		return err
 	}
 
+	var version string
+	var addr net.IPNet
+	var gateway net.IP
+	if netConfig.IPv6Address != nil {
```

Comment:
Why is this an "if / else" block? A task/pod should be able to have both an IPv4 and IPv6 address. Having an address-family-agnostic address array solves that problem.